Client/Frontend/Login/LoginInfoViewController.swift

```diff
@@ -417,7 +417,27 @@ extension LoginInfoViewController: LoginInfoTableViewCellDelegate {
       completion?(status)
     }
   }
+  
+  func didSelectCopyWebsite(_ cell: LoginInfoTableViewCell, authenticationRequired: Bool) {
+    func addPasswordToPasteBoardWithExpiry() {
+      let expireDate = Date().addingTimeInterval(5.minutes)
 
+      UIPasteboard.general.setItems(
+        [[UIPasteboard.typeAutomatic: cell.descriptionTextField.text ?? ""]],
+        options: [UIPasteboard.OptionsKey.expirationDate: expireDate])
+    }
```

Comment:
We should probably switch this to use our convenience method [`setSecureString(_:expirationDate:)`](https://github.com/brave/brave-ios/blob/41ddb78f1dd31a675d9d01ea969f115f8a911065/Sources/BraveWallet/Extensions/UIPasteboardExtensions.swift#L14) in a future PR (we'd have to move it out of BraveWallet first)

Reply:
Sounds good, I will keep in mind.